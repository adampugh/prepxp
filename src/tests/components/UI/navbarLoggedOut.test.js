import React from "react";
import { shallow } from "enzyme";

import NavbarLoggedOut from "../../../components/UI/navbarLoggedOut";


describe("<NavbarLoggedOut />", () => {
    it("should match snap", () => {
        const wrapper = shallow(<NavbarLoggedOut />);
        expect(wrapper).toMatchSnapshot();
    });
});