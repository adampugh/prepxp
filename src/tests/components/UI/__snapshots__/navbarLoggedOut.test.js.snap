// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarLoggedOut /> should match snap 1`] = `
<div>
  <div
    className="slideOut"
    id="drawer"
  >
    <Link
      replace={false}
      to="/prepxp/dashboard"
    >
      <h1>
        Home
      </h1>
    </Link>
    <Link
      replace={false}
      to="/prepxp/"
    >
      <h1>
        About
      </h1>
    </Link>
    <Link
      replace={false}
      to="/prepxp/blog"
    >
      <h1>
        Blog
      </h1>
    </Link>
    <Link
      replace={false}
      to="/prepxp/login"
    >
      <h1>
        Login
      </h1>
    </Link>
    <p>
      <i
        className="fab fa-twitter"
      />
      <i
        className="fab fa-facebook-f"
      />
      <i
        className="fab fa-youtube"
      />
    </p>
  </div>
  <div
    className="navbar"
  >
    <div
      className="container-l"
    >
      <div
        className="navbar__loggedOut"
      >
        <h1
          className="logo"
        >
          PrepXP
        </h1>
        <h1
          className=""
          onClick={[Function]}
        >
          â˜°
        </h1>
      </div>
    </div>
  </div>
</div>
`;
